plaid/src/apis/github/environments.rs

```diff
@@ -0,0 +1,213 @@
+use std::collections::HashMap;
+
+use serde::Serialize;
+use serde_json::Value;
+
+use alkali::asymmetric::seal;
+
+use crate::apis::{github::GitHubError, ApiError};
+
+use super::Github;
+
+#[derive(Serialize)]
+struct DeploymentBranchPolicy {
+    protected_branches: bool,
+    custom_branch_policies: bool,
+}
+
+#[derive(Serialize)]
+struct CreateEnvironmentPayload {
+    wait_timer: u16,
```

Comment:
What is this for? These structs might need some more documentation

Reply:
Added doc comments and a link to GH REST API